libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/MapboxMapCompassParams.kt

```diff
@@ -0,0 +1,62 @@
+package com.mapbox.navigation.dropin
+
+/**
+ * Params describing the map compass config.
+ *
+ * @param enabled true if the compass should be shown on the map, false otherwise
+ */
+class MapboxMapCompassParams private constructor(
```

Comment:
But if we allow custom drawable, our placeholders may stop working if the drawable size is different. Maybe for now we can just expose clickable and add a `compassImage` param when we can have compass as AndroidView and better control its positioning?

Reply:
In Drop-In UI we allow end users to change the look of other action buttons from rectangular to circular. By default our compass drawable will be rectangular to match other buttons. But if an end user changes all buttons to circular, and we don't allow to specify the image, the compass will remain rectangular. This will look weird. 

> our placeholders may stop working if the drawable size is different

What do you mean by stop working?

Reply:
> What do you mean by stop working?

I mean they will be useless as they have some fixed sizes. And if the drawable has a different size, the compass/scalebar may overlap over views.

Reply:
> And if the drawable has a different size, the compass/scalebar may overlap over views.

Shouldn't this be app developers responsibility to make sure that the image size is correct? I say it because, the placeholder containers don't contain the compass neither the scalebar. They are there so other views like action button and speed limit can constrain to it. If you look at 
```
android:id="@+id/speedLimitLayout"
android:layout_width="75dp"
android:layout_height="86dp"
```
and

```
android:id="@+id/actionListLayout"
android:layout_width="76dp"
android:layout_height="wrap_content" 
```
they both have a fixed width. We should assign the same fixed width to compass placeholder `Framelayout`. But in any case existence, position and size of these containers don't affect where the compass and scalebar will be placed or how they look.

Reply:
Do you mean that we should just specify in the docs that the compass should be 48 dp? Then OK, it would work fine.